### ISSUE TITLE
Detail information about pre-training dataset

### ISSUE THREAD

**Issue by rtanaka-lab** [shabie/docformer#20]
Thank you for the great work! I was wondering what criteria (ex. document words) were used to select the subset of 5 million pages from IIT-CDIP dataset in pre-training. To compare our model with docformer under the same condition, please share information or release a pre-process code.

**Comment by uakarsh** [comment]
Hi, if you meant about the IIT-CDIP dataset, it is taking lot of time for making the dataset and then saving it (since pytesseract is slow). So, authors would be releasing the dataset used in pre-trained method soon, so maybe we can wait till that time, for obtaining the performance of our implementation on the same dataset

If you are talking about the code, you can go through the following steps:
1. Import create_features, and apply it on each image and save it in pickle file
2. Create a dataset, which can read the pickle file and then return the features,target
3. Create a dataloader object, and then loop through each entry and apply the model
4. Backpropagate the loss

I hope this helps

**Comment by rtanaka-lab** [comment]
Looking forward to releasing the pre-training dataset!
